stage_descriptions/geospatial_6_ek6.md

```diff
@@ -0,0 +1,79 @@
+In this stage, you'll add support for calculating the distance between two locations.
+
+### The `GEODIST` command
+The `GEODIST` command returns the distance between two members. It can also take a optional last parameter which is the unit in which to express the distance in. The default unit is meters. The valid units are meters(m), kilometers(km), miles(mi), or feet(ft).
+The syntax for `GEODIST` command is:
+```
+GEODIST <key> <location1> <location2> [unit]
```

Comment:
Let's not mention unit here and just cover the default unit - we can mention units in the next stage explanation